test/create_test.go

```diff
@@ -37,24 +38,35 @@ func TestCreate(t *testing.T) {
 // BenchmarkCreate is a benchmark for the Create operation.
 func BenchmarkCreate(b *testing.B) {
 	for _, backendType := range []string{endpoint.SQLiteBackend, endpoint.DQLiteBackend} {
-		b.Run(backendType, func(b *testing.B) {
-			b.StopTimer()
-			g := NewWithT(b)
+		for _, workers := range []int{1, 2, 4, 8, 16} {
```

Comment:
Could you please explain the reasoning behind using `2^(n−1)`?
Is this the ideal range we should use? 
Admittedly, it's a bit difficult to see if this is the ideal range in the benchmark against main without changes- maybe we can tweak these at a later stage if we find it necessary.

Reply:
Ah that makes sense, I thought you might have a good answer! Wow, AWS can even give you 64 cores but I think that might go beyond our regular use case xD